src/main/fc/fc_msp.c

```diff
@@ -1582,6 +1582,21 @@ static bool mspFcProcessOutCommand(uint16_t cmdMSP, sbuf_t *dst, mspPostProcessF
         break;
 #endif
 
+#ifdef USE_RATE_DYNAMICS
+
+    case MSP2_INAV_RATE_DYNAMICS:
+        {
+            sbufWriteU8(dst,  currentControlRateProfile->rateDynamics.sensitivityCenter);
```

Comment:
ocd comment. There are some extra spaces in some lines :)

Reply:
not anymore ;)